packages/frontend-2/components/viewer/anchored-point/NewThread.vue

```diff
@@ -72,6 +72,7 @@
       </ViewerCommentsPortalOrDiv>
     </div>
   </div>
+  <div v-else></div>
```

Comment:
Why is this needed?

Reply:
If shouldShowThreadBubble returns false, the component will still be mounted, but there will be nothing inside the , this can cause side effects, and has done elsewhere in the app. Now, when I see this, I add an empty div so something will still be rendered and avoid those side effects. There was no side effect that I noticed here, but it's just good practice.